src/scss/_list.scss

```diff
@@ -125,15 +150,13 @@
   width: 1.3rem;
 
   &:checked {
-    & + .list-desc {
+    & ~ .list-desc {
       color: var(--main-grey);
 
       &::before {
-        opacity: 1;
-        width: 100%;
-        height: 2px;
-        border-radius: 10px;
-        top: 50%;
+        position: absolute;
+        // opacity: 1;
```

Comment:
- [ ] **Optional:** It is a professional practice to remove all unused or commented code before committing. Kindly remove it in future projects.